docs/conf.py

```diff
@@ -264,6 +264,12 @@
 
 exclude_patterns = ['api/*', 'links.rst']
 
+# Intersphinx Extension Configuration
+
 intersphinx_mapping = {
     'python': ('https://docs.python.org/3', None),
+    "glossary": (
+        f"https://docs.openedx.org/en/latest/developers/references/developer_guide/glossary.html",
```

Comment:
I want to make sure I've done this right, in this commit: https://github.com/openedx/XBlock/pull/686/commits/437e743c96f0368e672de628a822181775036064

Reply:
I clearly didn't do this right as I'm getting this error:

`intersphinx inventory 'https://docs.openedx.org/en/latest/developers/references/developer_guide/glossary.html/objects.inv' not fetchable due to <class 'requests.exceptions.HTTPError'>: 404 Client Error: Not Found for url: https://docs.openedx.org/en/latest/developers/references/developer_guide/glossary.html/objects.inv`